docs/sources/features/datasources/loki.md

```diff
@@ -38,10 +38,7 @@ Just add it as a data source and you are ready to query your log data in [Explor
 
 ## Querying Logs
 
-Querying and displaying log data from Loki is available via [Explore](/features/explore).
-Select the Loki data source, and then enter a log query to display your logs.
-
-> Viewing Loki data in dashboard panels is not supported yet, but is being worked on.
+Querying and displaying log data from Loki is available via [Explore](/features/explore), and with [Table panel](/features/panels/table_panel/) in dashboards. Select the Loki data source, and then enter a log query to display your logs.
```

Comment:
Should be the logs panel now, no?

Reply:
Not sure is it still alpha? If so I am not sure if it should be mentioned here or not.

Reply:
Good point, let's change this part in the PR that brings this out of alpha.